### PR TITLE
feat: Update infos when menu is collapsed - MEED-1809 - Meeds-io/meeds#754

### PR DESCRIPTION
**Pull request by SaraBoutej**
Fix feedback

### CODE REVIEW

File: webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/HamburgerMenuNavigationButton.vue [hunk 1]
```diff
@@ -39,6 +39,9 @@
 </template>
 <script>
 export default {
+  data: () => ({
+    spaceUnreadItems: null
+  }),
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/HamburgerMenuNavigationButton.vue]: data must go after props definition

File: webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/HamburgerMenuNavigationButton.vue [hunk 2]
```diff
@@ -69,21 +81,34 @@ export default {
...
-        if (spaceId && this.unreadPerSpace && this.unreadPerSpace[spaceId]) {
-          this.unreadPerSpace[spaceId] ++;
-        } else {
-          this.unreadPerSpace[spaceId] = 1;
-        }
-      }  else if (wsEventName === 'notification.read.item') {
-        if (spaceId && this.unreadPerSpace && this.unreadPerSpace[spaceId] > 0) {
-          this.unreadPerSpace[spaceId] --;
+      if (!this.spaceUnreadItems) {
+        this.unreadPerSpace = {};
```
Comment (boubaker) [inline webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/HamburgerMenuNavigationButton.vue]: changin a props isn't recommended

Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/HamburgerMenuNavigationButton.vue]: Done!

File: webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/HamburgerMenuNavigationButton.vue (hunk 1 above)
Comment (SaraBoutej) [reply webapp/portlet/src/main/webapp/vue-apps/hamburger-menu/components/HamburgerMenuNavigationButton.vue]: Done!

**Review by boubaker — DISMISSED** [review]

**Pull request closed without merging**